tests/setup_environment_test/setup_environment_test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from sqlalchemy.exc import OperationalError
+
+from src.digitaltwin.setup_environment import get_connection_from_profile
+
+
```

Comment:
This needs a docstring! :) What is the point of the test? How is it testing this? It would be worth thinking what is required for Pooja to finish this off :) 

Otherwise it looks good! I particular like how you've targeted this small but important test of the overall functionality.